paddle/framework/operator.cc

```diff
@@ -12,11 +12,68 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
 
+#include <algorithm>
+
 #include "paddle/framework/operator.h"
 
 namespace paddle {
 namespace framework {
 
+void OperatorBase::CreateArgumentOffsetMap(const OpProto& proto) {
```

Comment:
Argument is caffe2 has the same meaning with Attribute in Paddle. Should we name it CreateInOutOffsetMap or something else?